### PR TITLE
Doc: Corrected build instructions

### PR DESCRIPTION
**Pull request by relativisticelectron**
Fix for https://github.com/cryptoadvance/specter-desktop/issues/1935

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -95,7 +95,6 @@ pip3 install virtualenv
 virtualenv --python=python3 .env 
 source .env/bin/activate
 pip3 install -r requirements.txt --require-hashes
-pip3 install -e .
 python3 setup.py install # also compiles the babel translation-files
```
Comment (k9ert) [inline docs/development.md:98]: I would have thought that the solution here is to remove the `python3 setup.py install` but keep `pip3 install -e .`?! After all the compilation of the babel translation-files is not necessary for a dev-env, imho?!

Comment (relativisticelectron) [reply docs/development.md:98]: Yes it turns out that keeping `pip3 install -e .`  without `python3 setup.py install` also works (I didn't realize that).   I would prefer however build instructions that include babel instructions.  What if someone tests translation issues?

Comment (moneymanolis) [reply docs/development.md:98]: Plus one for `pip3 install -e . `- works much more reliably. @relativisticelectron if you find someone who tests translation issues I will personally guide this person through his dev setup ;-)

**Review by moneymanolis — CHANGES_REQUESTED** [review]
Keep `pip3 install -e .`

**Comment by relativisticelectron** [comment]
> Keep `pip3 install -e .`

Done.

**Review by moneymanolis — APPROVED** [review]

**Pull request merged**